### PR TITLE
Adds documentation + start V4 VCL

### PR DESCRIPTION
**Pull request by toonvd**
V4 VCL is not complete yet but documentation can be merged already

### CODE REVIEW

File: docs/CREATING_A_VCL.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      "first_byte_timeout": "6000s",
+      "connect_timeout": "6000s"
+    }
+  },
+  "default_backend_option": {
+    "first_byte_timeout": "30s",
+    "connect_timeout": "5s",
+    "between_bytes_timeout": "2s"
+  },
+  "probe_url": "YOURURLFORHEALTCHECKHERE",
```
Comment (IvanChepurnyi) [inline docs/CREATING_A_VCL.md]: I think you can add a comment that this is optional, as health check is required only for load balancing.

Comment (toonvd) [inline docs/CREATING_A_VCL.md]: Will correct this together with the completely tested v4 VCL

Comment (IvanChepurnyi) [inline docs/CREATING_A_VCL.md]: Thanks! Looking forward to it!

**Comment by IvanChepurnyi** [comment]
Hi Toon,

I am merging your pull request to `develop` branch and will fix VCL for Varnish 4 later myself. Thanks for your contribution!

**Comment by toonvd** [comment]
Hi, no problem, time is an issue here, and have not needed to implement Varnish on Mage 1 anymore :)

**Pull request merged**